### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by bastelfreak**
this adds new infos to the docs about the DHCP endpoint

### CODE REVIEW

**Comment by bastelfreak** [comment]
this implements https://github.com/virtapi/marmoset/issues/24

File: README.md [hunk 1]
```diff
@@ -24,10 +25,26 @@ Monkeying around with virtual machines and pxe configs.
 
 ## Setup
 
-Create `marmoset.conf` before using marmoset! See `Configuration` for details.
+Clone the repo, for example into /opt. Than copy the service file into the systemd directory and reload systemd to recognize the file:
```
Comment (taxus13) [inline README.md]: "Clone the repository into a directory of your choice, /opt for example" , "then"

**Comment by bastelfreak** [comment]
this should be merged after https://github.com/virtapi/marmoset/pull/31 because it adds readme for this PR

File: README.md [hunk 2]
```diff
@@ -24,10 +25,26 @@ Monkeying around with virtual machines and pxe configs.
...
+cd /opt
+git clone https://github.com/virtapi/marmoset.git
+cd marmoset
+cp ext/marmoset.service /etc/systemd/system/
+systemctl daemon-reload
+```
 Copy the `marmoset.conf.example` to `marmoset.conf` and adjust the settings to your needs.
 Checkout the Comments in the file our our [Configuration](#configuration) section.
 
+Now we need to setup a virtualenv and install the requirement python packages (remove libvirt from the requirements.txt and `pkg-config libvirt gcc` from the list of packages to install if you don't want to manage VMs with marmoset):
```
Comment (taxus13) [inline README.md]: "and install the required python packages"

File: marmoset.conf.example [hunk 3]
```diff
@@ -34,9 +35,13 @@ Network:
 # storage name is given.
 Storage:
 
+# this gets used for storing PXE options
```
Comment (taxus13) [inline marmoset.conf.example]: "this will be used for"

File: marmoset.conf.example [hunk 4]
```diff
@@ -34,9 +35,13 @@ Network:
...
 [PXEConfig]
 ConfigDirectory: /srv/tftp/pxelinux.cfg/
 
+# this gets used for storing installimage options
```
Comment (taxus13) [inline marmoset.conf.example]: "this will be used for"

File: README.md (hunk 1 above)
Comment (bastelfreak) [inline README.md]: fixed

File: README.md (hunk 2 above)
Comment (bastelfreak) [inline README.md]: fixed

File: marmoset.conf.example (hunk 4 above)
Comment (bastelfreak) [inline marmoset.conf.example]: fixed

File: marmoset.conf.example (hunk 3 above)
Comment (bastelfreak) [inline marmoset.conf.example]: fixed

**Comment by bastelfreak** [comment]
in my opinion this is ready for merge now. but we should wait until we merge https://github.com/virtapi/marmoset/pull/31

**Comment by bastelfreak** [comment]
:+1: from @sebastianrakel on IRC

**Pull request merged**